data/library/metadata.json

```diff
@@ -7,7 +7,8 @@
         "work_title": "Iliad",
         "work_urn": "urn:cts:greekLit:tlg0012.tlg001:",
         "type": "edition",
-        "first_passage_urn": "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7"
+        "first_passage_urn": "urn:cts:greekLit:tlg0012.tlg001.perseus-grc2:1.1-1.7",
+        "citation_scheme": ["book", "line"]
```

Comment:
I'll make sure to carry this over to work done on ![](https://github.trello.services/images/mini-trello-icon.png) [Ingest texts into svm-atlas](https://trello.com/c/ox0bIENs)